test/unit/select_comparators_test.lua

```diff
@@ -139,3 +141,24 @@ g.test_unicode_collations = function()
     t.assert(not func_le_unicode({'A', 'Á', 'Ä'}, {'a', 'A', 'a'}, 3))
     t.assert(func_le_unicode_ci({'A', 'Á', 'Ä'}, {'a', 'A', 'a'}, 3))
 end
+
+g.test_uuid_type_parts = function()
+    local key_parts = {
+        {type = 'uuid'}, {}
+    }
+
+    local func_eq, err = select_comparators.gen_func(operators.EQ, key_parts)
+    t.assert(err == nil)
+    local u1, u2 = uuid.fromstr('8a1ffbc0-241e-11eb-8d27-0800200c9a66'), uuid.fromstr('8f1ffbc0-241e-11eb-8d27-0800200c9a66')
```

Comment:
Why don't you split it into 2 lines?

Reply:
Personally I don't see any difference between this values without detail reading. It's better to use something like "aaaaaaaa-aaaa-4000-b000-000000000001" and "bbbbbbbb-bbbb-4000-b000-000000000001"